client/src/nl/sensorlab/videowall/animation/baseanimations/bouncypixels/BouncyPixelsAnimation.java

```diff
@@ -0,0 +1,55 @@
+package nl.sensorlab.videowall.animation.baseanimations.bouncypixels;
+
+import java.util.ArrayList;
+import com.cleverfranke.util.PColor;
+import nl.sensorlab.videowall.animation.BaseAnimation;
+import processing.core.PApplet;
+import processing.core.PGraphics;
+
+public class BouncyPixelsAnimation extends BaseAnimation{
```

Comment:
This class feels like it's just a wrapper around BouncyPixelSystem. I think you can merge the whole BouncyPixelSystem in this class without losing any clarity.